features/step_definitions/borrower_views_deed.rb

```diff
@@ -66,3 +66,7 @@
     charge.'
   )
 end
+
+Then(/^the property address is displayed on the deed$/) do
+  page.should have_content('1 High Street, High Wycombe, CR0 6TY')
```

Comment:
I would refer to the address set in the @deed data object when checking the page rather than checking for a hardcoded value. So something like page.should have_content(@deed.address), then if the address is changed in the deed this will pick it up rather than having to change it in both the deed data and the steps that reference it if this makes sense?